.gitmodules

```diff
@@ -13,3 +13,10 @@
 [submodule "packages/contracts-bedrock/lib/forge-std"]
 	path = packages/contracts-bedrock/lib/forge-std
 	url = https://github.com/foundry-rs/forge-std
+[submodule "packages/contracts-bedrock/lib/safe-contracts"]
+	path = packages/contracts-bedrock/lib/safe-contracts
+	url = https://github.com/safe-global/safe-contracts
+  branch = v1.4.0
```

Comment:
The commit `bf943f` corresponds to `v1.4.1` it seems, see the file below `lib/safe-contracts`

Reply:
I'm not sure how that commit is relevant. If you'd prefer 1.4.1 LMK and I'll bump this.

Reply:
If that's the intent, please approve https://github.com/ethereum-optimism/optimism/pull/7299 and close this one. :)

Reply:
OK, [the submodule](https://github.com/ethereum-optimism/optimism/pull/7293/files#diff-88425826d202bdcd3a5e4d48a8d23f29584966c42cbe5db3d29657368bbfa8c9) is now at the commit of [v1.4.0](https://github.com/safe-global/safe-contracts/releases/tag/v1.4.0) (`e870f514ad34cd9654c72174d6d4a839e3c6639f`).